x-pack/plugins/security_solution/public/management/components/endpoint_execute_action/execute_action_host_response.tsx

```diff
@@ -39,25 +39,17 @@ export const ExecuteActionHostResponse = memo<ExecuteActionHostResponseProps>(
       [action.outputs, agentId]
     );
 
-    const isTruncatedFile = useMemo<boolean>(
-      () =>
-        (outputContent?.output_file_stderr_truncated ||
-          outputContent?.output_file_stdout_truncated) ??
-        false,
-      [outputContent]
-    );
-
     return (
       <>
         <EuiFlexItem>
           <ResponseActionFileDownloadLink
             action={action}
             buttonTitle={EXECUTE_FILE_LINK_TITLE}
             canAccessFileDownloadLink={canAccessFileDownloadLink}
-            isTruncatedFile={isTruncatedFile}
             data-test-subj={`${dataTestSubj}-getExecuteLink`}
             textSize={textSize}
           />
```

Comment:
`ResponseActionFileDownloadLink` component's `isTruncatedFile` prop is never used - should we remove it? (I'm not familiar with the development plan, so not sure whether it'll be needed)